### ISSUE TITLE
Consultas varias Heladas y Congelacion

### ISSUE THREAD

**Issue by ANeuensc** [IIC2233/Syllabus#239]
* [X] Leí las reglas del foro (https://github.com/IIC2233/Syllabus/wiki/3.-Reglas-del-foro-(issues)).
* [X] Revisé el compilado de dudas de la tarea y no encontré una *issue* similar a la mía ([click aquí para ir al compilado](/IIC2233/Syllabus/issues/11)).
* [X] Mi duda no se trata sobre una librería o *built-in*, ya que eso se pregunta en la *issue* creada para ello ([click aquí para ir a esa _issue_](/IIC2233/Syllabus/issues/10)).
* [X] De ser necesario, colocaré código simple que permita explicar mi problema o duda, ya que compartir código de esta evaluación puede ser considerado una falta a la ética.

### Duda

3 Consultas del mismo tipo:
El enunciado dice que cuando hay una Helada y la planta tiene punto de congelación mayor que la temperatura, entonces esta se congela, mi consulta es si una planta se congela CUALQUIER día que tenga temperatura menor al punto de congelación, o solo durante una Helada.
En caso de que Se haya congelado una planta en una Helada y al siguiente día no haya Helada pero la temperatura sigue más baja que el punto de congelación, ¿se tiene que mantener congelada la planta?
Por ultimo, si planto una planta nueva desde el inventario que tiene un punto de congelación mayor que la temperatura actual, ¿Se congela instantaneamente?

**Comment by R3Y3X** [comment]
holaaaaaaa espero que estés super 😄.
1) Se puede congelar en CUALQUIER momento tanto día normal como en Helada.
2)Si, ya que para que una planta esté congelada basta con que su resistencia térmica sea mayor que la temperatura del jardín.
3)en esta tarea se busca evaluar (además de los contenidos) como se enfrentan a distintas problemáticas, así que eres libre de escoger si se congela instantáneamente, recuerda que cada suposición que hagas debes dejarla clara en el README.
Éxito en tu rarea ⚡

**Comment by mrebolledo03** [comment]
Entonces el evento de helada solo hace que baje la temperatura del jardin, no significa que solo en un dia con helada se puedan congelar? Si un dia cualquiera la temperatura es menor a la de su resistencia termica igual se congela la planta?

**Comment by R3Y3X** [comment]
@mrebolledo03 sí, un día cualquiera se puede congelar la planta si la temperatura del jardín es menor a la resistencia térmica de la planta
